cognite_toolkit/modules/cdf_data_pipeline_asset_default/default.config.yaml

```diff
@@ -0,0 +1,14 @@
+# This is the configuration file used for the data realted to asset data modeling 
+---
+# spesify the site/asset location where data comes from, ex 'valhall' or if they are generic for all assets use 'all' 
+location_name: <update-me> 
```

Comment:
We need to discuss site vs location today. It's a bit unclear to me how the infield location relates to the on-going site configuration work.